next permutation in c++

```diff
@@ -0,0 +1,69 @@
+#include <bits/stdc++.h>
+using namespace std;
+
+//to print a vector
```

Comment:
Function comments should be descriptive, i.e. "Prints a vector".
See https://google.github.io/styleguide/cppguide.html#Function_Comments.

You should also properly format the comment and use punctuation/capitalization:
```c++
// Prints a vector.
```